src/test/common/steps/common-steps.js

```diff
@@ -117,19 +117,19 @@ async function enterEmailAddressAndSubmit (emailAddress = EMAIL_ADDRESS) {
   }
 }
 
-async function selectNoOnChildrenThreeOrYoungerPage () {
+async function selectNoOnChildrenPage () {
```

Comment:
This should probably be `selectNoOnDoYouHaveChildrenPage`. Similarly with the yes equivalent below.